apple/internal/codesigning_support.bzl

```diff
@@ -60,6 +60,7 @@ def _codesign_args_for_path(
         path_to_sign,
         platform_prerequisites,
         provisioning_profile,
+        codesignopts,
```

Comment:
Nit: can this be moved up?

Reply:
What ordering where you hoping for? I felt like there wasn't a super clear order to go for so I didn't try for one

Reply:
ah this group is alphabetical, I'll move